tests/ClientTest.cc

```diff
@@ -305,6 +305,7 @@ TEST(ClientTest, testCloseClient) {
         auto t0 = std::chrono::steady_clock::now();
         while ((std::chrono::steady_clock::now() - t0) < std::chrono::microseconds(i)) {
         }
+        sleep(1);
```

Comment:
Please move it into another PR if you want to fix the flaky test. BTW, sleeping 1 second is unacceptable here because it's in a 1000 times loop

Reply:
Oh, sorry. This was written by my local test and I submitted it by mistake.